homeassistant/components/home_connect/__init__.py

```diff
@@ -117,24 +117,22 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Set up Home Connect component."""
     hass.data[DOMAIN] = {}
 
-    if DOMAIN not in config:
-        return True
```

Comment:
This caused this method to return early and thus also not to set up its services.